src/mpid/ch4/shm/src/topotree.c

```diff
@@ -230,21 +236,20 @@ static void gen_tree_sharedmemory(int *shared_region, MPIDI_SHM_topotree_t * tre
  * over the different data-structures:
  * k_val : the tree K-value
  * shared_region : the shared memory region where the tree will be generated
- * max_entries_per_level : the maximum number of ranks per level
+ * num_packages : the maximum number of ranks at package_level
```

Comment:
the comment talking about ranks leaves me a but confused, still. should we just change it to "the number of packages"?

Reply:
Fixed.